.github/environment.yml

```diff
@@ -8,12 +8,13 @@ dependencies:
   - pyyaml
   - numpy
   - scipy
-  - camb=1.3.6
-  - isitgr
+  - camb
+  #- isitgr
```

Comment:
Why did you leave this commented out?

Reply:
I will remove it. I will also need to remove the -v arg in pip install before finalizing this PR.

Reply:
OK great. I was not opposed to leaving the -v in, so up to you. I think we can merge after that, so I'll approve now so you don't have to wait for me.